SpatialGDK/Documentation/content/dynamic-typebindings.md

```diff
@@ -0,0 +1,29 @@
+# Dynamic Typebindings
+To allow Unreal to replicate through the SpatialOS network stack and combine multiple dedicated server instances across one seamless game world, the GDK has to adapt Unreal networking functionality to work with the [SpatialOS Worker API (SpatialOS documentation)](https://docs.improbable.io/reference/latest/capi/introduction). It does  this seamlessly using `Dynamic Typebindings`. `Dynamic Typebindings` operate at [runtime]({{urlRoot}}/content/glossary#spatialos-runtime) so that your development iteration speed is not affected, despite your network code running on a completely different representation than Unreal’s.
```

Comment:
Maybe we should mention the `TypeBindingManager` to and a little about how we generate them.

Reply:
I feel it should go in the glossary too in the GDK terms section.